intranet/apps/preferences/fields.py

```diff
@@ -25,7 +22,5 @@ def to_python(self, value):
         return value
 
     def widget_attrs(self, widget):
-        attrs = {}
-        if self.max_length is not None:
-            attrs.update({'maxlength': str(self.max_length)})
+        attrs = {"maxlength": "17"}
```

Comment:
The maximum the regex will accept is 17, I'll add a commit in later.
We should standardize the phone fields and make them all display either "None" or be blank when they are empty. Currently, some of them display "None" when the field is empty and others are blank. I personally think that making them all blank is the best option.